corehq/apps/sms/change_publishers.py

```diff
@@ -1,10 +1,20 @@
 from corehq.apps.change_feed import topics
```

Comment:
sorry the diff doesn't make sense but it was on the commit. I meant not publishing in tests.